src/App.js

```diff
@@ -1,23 +1,47 @@
-import logo from './logo.svg';
+import { useState } from 'react';
 import './App.css';
+import foods from "./foods.json";
+import FoodBox from './components/FoodBox';
+import AddFoodForm from './components/AddFoodForm';
+import Search from './components/Search';
 
 function App() {
+  const [foodList, setFoodList] = useState(foods);
+
+  const createNewFoodItem = (newItem) => {
+    const updatedFoodList = [newItem, ...foodList];
+    setFoodList(updatedFoodList);
+  }
+
+  const searchFilter = (chars) => { 
+    let filteredList = foodList.filter((foodItem) => { 
+      return foodItem.name.toLowerCase().includes(chars.toLowerCase());
+    });
+    setFoodList(filteredList);
+  };
+
+  const deleteFoodItem = (foodId) => {
+    const allFoods = [...foodList];
+
+    const updatedFoods =  allFoods.filter((foodItem) => foodItem.id !== foodId);
+
+    setFoodList(updatedFoods);
+  }
+
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <AddFoodForm createNewFoodItem={createNewFoodItem}/>
+    <Search searchFilter={searchFilter}/>
+      <h1>Food List</h1>
+
+      {!foodList.length && <div><h1>No content</h1></div>}
+
+      {foodList.map((foodItem) => (
+        <div>
+          <FoodBox key={foodItem.id} foodItem={foodItem} deleteFoodItem={deleteFoodItem}/>
+        </div>
+      ))}
+
```

Comment:
You can use the component "Row", from Ant Design, and setup up a gutter. Also, remove the div from "FoodBox"

```suggestion
 <Row className="rows" gutter={[16, 16]}>
        {!foodList.length && ( <div><h1>No content</h1></div>)}

        {foodList.map((foodItem) => (
          <FoodBox key={foodItem.id} foodItem={foodItem} deleteFoodItem={deleteFoodItem}/>
        ))}
      </Row>
      ))}

```